alfresco-connector/config/_dev_axonivy/variables.yaml

```diff
@@ -1,12 +0,0 @@
-Variables:
-  # Custom variables of Alfresco Content Services API for environment dev_axonivy used for testing
-  Alfresco:
```

Comment:
why this has worked with environments? the name was different to name of the real service.

Reply:
fun fact, the variables were not really consumed, but just hardcoded unto the rest client. So I think didn't really work with the vars at all.